packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -103,181 +97,162 @@ describe('Button', () => {
+    beforeEach(() => browser.url('/#/raw/tests/button/long-text-button'));
 
-    Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
+    Terra.should.beAccessible({ context: '#long-text-block' });
+    Terra.should.matchScreenshot({ selector: '#long-text-block' });
   });
 
   describe('Button Types', () => {
-    describe('Type button', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/with-type-button'));
+    beforeEach(() => browser.url('/#/raw/tests/button/button-types'));
+    Terra.should.beAccessible();
 
-      Terra.should.beAccessible();
-      Terra.should.matchScreenshot({ selector: 'button' });
+    describe('Type button', () => {
+      Terra.should.matchScreenshot({ selector: '#buttonWithTypeButton' });
 
-      Terra.should.themeEachCustomProperty(
-        'button',
-        {
-          '--terra-button-text-only-horizontal-margin': '20px',
-        },
-      );
+      Terra.should.themeEachCustomProperty('#buttonWithTypeButton', {
+        '--terra-button-text-only-horizontal-margin': '20px',
+      });
     });
 
     describe('Type reset', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/with-type-reset'));
-      Terra.should.beAccessible();
-      Terra.should.matchScreenshot({ selector: 'button' });
+      Terra.should.matchScreenshot({ selector: '#buttonWithTypeReset' });
     });
 
     describe('Type submit', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/with-type-submit'));
-      Terra.should.beAccessible();
-      Terra.should.matchScreenshot({ selector: 'button' });
+      Terra.should.matchScreenshot({ selector: '#buttonWithTypeSubmit' });
     });
   });
 
   describe('Button with Icon', () => {
-    describe('Icon and Text', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/icon-neutral'));
+    beforeEach(() => browser.url('/#/raw/tests/button/icon-button'));
+    Terra.should.beAccessible();
 
-      Terra.should.beAccessible();
+    describe('Icon and Text', () => {
       Terra.should.matchScreenshot({ selector: '#iconNeutralButton' });
 
-      Terra.should.themeEachCustomProperty(
-        '#iconNeutralButton',
-        {
-          '--terra-button-text-and-icon-horizontal-margin': '20px',
-          '--terra-button-text-and-icon-margin-between': '20px',
-        },
-      );
+      Terra.should.themeEachCustomProperty('#iconNeutralButton', {
+        '--terra-button-text-and-icon-horizontal-margin': '20px',
+        '--terra-button-text-and-icon-margin-between': '20px',
+      });
     });
 
     describe('Icon and Text Reversed', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/icon-reversed'));
-
-      Terra.should.beAccessible();
       Terra.should.matchScreenshot({ selector: '#iconReversedButton' });
     });
 
     describe('Icon only', () => {
-      beforeEach(() => browser.url('/#/tests/button-tests/icon-only'));
-
-      Terra.should.beAccessible();
```

Comment:
How come the accessible tests are removed here? Can't see where they are added back?

Reply:
@JakeLaCombe We don't need to check accessible for each of these anymore cause these examples were combined into one page and the accessible testing for this page is done here: https://github.com/cerner/terra-core/pull/1312/files/f72f30468612ccda45ae50f85fc52012b7f1a239#diff-855d5add99fb4ef5ee0898c78843b5bfR228

https://terra-core-deployed-pr-1312.herokuapp.com/static/#/tests/button/icon-button